app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager.rb

```diff
@@ -59,4 +65,8 @@ def self.ems_type
   def self.description
     @description ||= "IBM Power Systems Virtual Servers".freeze
   end
+
+  def region
+    provider_region.empty? ? "us-south" : provider_region
+  end
```

Comment:
This isn't needed because you're handling it in the collector here https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/30/files#diff-53f571cf57b42afefdd9b8a927cff8beR50-R52

Reply:
removed